Add notes for 03-11-2019 TSC meeting (#695)
imageworks/OpenColorIO: docs/tsc/meetings/2019-03-11.md

Mar 11, 2019

Host: Michael Dolan

Rotating Secretary: Carl Rand

Attendees:
  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic

>   * [ ] Sean Cooper (_TSC_) - DNEG

Reviewer: ```suggestion
  * [X] Sean Cooper (_TSC_) - DNEG
```

Sean was present.

Author: Correction made in commit 519694f4b44c3d9a5e57f5084f221329b4734236

  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
  * [X] Thanh Ha - Academy Software Foundation / Linux Foundation
  * [X] Patrick Hodoul (_TSC_) - Autodesk
  * [X] Bernard Lefebvre - Autodesk
  * [X] John Mertic - Academy Software Foundation / Linux Foundation
  * [X] Carl Rand (_TSC_) - Foundry
  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
  * [X] Kevin Wheatley (_TSC_) - Framestore

# **OCIO TSC Meeting Notes**

* Prerequisites to moving the repository to ASWF
    - Resolve "ext/" dependencies prior to the move.
    - What are we going to do with the "ext/" directory?

>     - Do we maintained or replace?

Reviewer: ```suggestion
    - Do we maintain or replace?
```

Author: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

    - We will need to make the TAC aware of any external code that is needed when we ship with the library.

* CLA/DCO requirements
    - Needs them signed before doing merges into the new repository.
    - John: Resolving issues with the initial setup. Team is currently backlogged on other projects but hoping to get something done in the next couple of days.
    - DCO when we are ready to DCO sign off with commits.
    - Every commit will have DCO sign off.
    - John: Some info can be pulled or referenced from the TAC guidelines.
    - Larry: Who needs to sign off the work?
        - John: We can refer to ​https://developercertificate.org/
    - Michael: This isn’t required until we move to the new account. Not to worry about it for now.
    - Want to have the Linux foundation CLA system up and running first, pre-sign all the CLAs, and do one commit to flick the switch.
    - Larry: Have agreed on the CLA but not on all the surrounding areas e.g. signature blocks etc.

> 

Reviewer: Could add:

`- **TODO**: Document DCO sign off practices in CONTRIBUTING.md`

Author: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

* Googlegroup archiving & updated gh-pages deployment
    - Announced we will pull down the group in a week.

>     - A script will be run to get old contend and re-import it into the new group.

Reviewer: ```suggestion
    - A script will be run to get old content and re-import it into the new lists.
```

Author: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236

    - John: Test run will done first to see how it works.
    - The OCIO website will be updated accordingly.
    - The new website has been updated but is yet to be deployed.

* Fixed functions in the repository
    - Sean asked if there has been any more follow up conversations about this.
    - Kevin thought that this wasn't a show stopper for him.
    - Kevin raised the camera log transform and whether or not the goal is being set from the ACES requirements.
    - May require to change the camera op to support ARRI.
    - **TODO**: Post a review as it's been up for a couple of weeks now.

* Next 1.1.x release
    - There are a small number of commits that will need to be signed off before the release is made.
    - Agreed to call it the 1.1.1 release.
    - It will be the first release of the year.
    - **TODO**: Michael to finalize things to push it forward.
    - **TODO**: Will need to update the change log for the new version.

* Items for next TSC meeting agenda:

>     - xxx

Reviewer: If there's nothing here, we can delete this section.

Author: Correction made in 519694f4b44c3d9a5e57f5084f221329b4734236
